src/aiodynamo/models.py

```diff
@@ -174,11 +177,36 @@ def default(cls):
             await asyncio.sleep(self.retry_delay)
 
 
-@dataclass(frozen=True)
-class ThrottleConfig:
-    max_attempts: int
-    delay_func: Callable[[int], Timeout]
-
+class ThrottleConfig(metaclass=abc.ABCMeta):
     @classmethod
     def default(cls):
-        return ThrottleConfig(5, lambda attempt: attempt + 1)
+        return DecorrelatedJitterThrottling()
+
+    @abc.abstractmethod
+    async def attempts(self) -> AsyncIterable[None]:
+        raise NotImplementedError()
+
+
+@dataclass(frozen=True)
+class DecorrelatedJitterThrottling(ThrottleConfig):
+    time_limit_secs: int = 60
+    max_time_secs: int = 60
+    base_delay_secs: int = 0.05
+    max_delay_secs: int = 1
```

Comment:
I think we need to pass something to Deco, so that it knows how much we're trying to do:
* inconsistent read costs ½RCU
* [consistent read]
* write costs 4RCU or something
* query costs anywhere between 125RCU (1MB/4K/2) to 100KRCU (tiny items)?
--
* table could be provisioned however...
* there could be N competing users (e.g. containers)
--
I don't have a good idea how to predict how long to wait for dynamo; but I figure that get and scan should have different initial conditions...

Reply:
Then again, it's nicely exponential, with ╳1.5 per iteration, so it only takes ~8 steps to get from .05s to 1s, should be OK?

Reply:
what is `Deco`?

Reply:
your decompressed random throttler thing... DecorrelatedBlahBlahThrottle.